src/style.css

```diff
@@ -1,3 +1,87 @@
+:root {
+  --active-link: #00acdc;
+  --text-color: #000;
+  --bg-color: #fff;
+  --shadow-color: 0px 2px 4px 0px #0000000d;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
```

Comment:
Consider adding a fallback font-family in case 'Roboto' doesn't load. For example, 'Arial' as a sans-serif fallback would be a good choice.